### ISSUE TITLE
A11y_Windows Template Studio_Change Summary_Screenreader: Screenreader reads “Modified Files” button as a group that is not correct announcement.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/WindowsTemplateStudio#3891]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#GH_WindowsTemplateStudio_Oct2020;#Windows Template Studio;#Win10;#FTP;#STP;#DesktopApp;#WCAG1.3.1;#NVDA;#Narrator;#Screenreader;#Benchmark;#A11y_Windows Template Studio;

### **Environment Details:**

Application Name: Windows Template Studio (WinTS)
Templates version: 3.8.20273.1
Wizard version: 3.8.20273.1Date: 2020-06-17T21:13:20.174Z

Microsoft Visual Studio Enterprise 2019 Preview
Version 16.8.0 Preview 3.1
Windows Version: Windows10
NVDA Version: 2019.3.1

### **Repro Steps:**

1.	Open Visual Studio 2019 
2.	Click on File from menu and then select new and then project.
3.	Select the "Windows template studio (WPF.Net Core)".Click on next
4.	"Configure your new project" screen will appear. Click on create button.
5.	New WPF App screen will open. Navigate to "Services" in the left navigation menu and click on it.
6.	"Add Services "screen will appear.
7.	Click on Create button
8.	Visual studio main page will get open.
9.	Navigate to solution explorer, navigate to your app name, and then press right click.
10.	Navigate to window template studio and select new page.
11.	New page screen will appear. Navigate to "Changes Summary"  
12.	Start Narrator and navigate to  “Modified Files” button with the help of Keyboard tab key then hit Enter.

### **Actual:**

Screen reader reads “Modified Files” button as a group that is not correct announcement.

**Same issue is also observed with “New Files” button present in this scenario.**

**Same issue is also observed with NVDA.**

### **Expected:**
Narrator should announce “Modified Files” as a button not group.

### **User Impact:**
User will not get correct information if narrator announce group instead of Button.

**Comment by sibille** [comment]
Verified in dev-nightly version:
Wizard version: 0.22.20310.1

You can install the dev-nightly version from our nightly feed (https://www.myget.org/F/windows-template-studio-nightly/vsix/).
For instructions on how install WTS from the dev-nightly feed in Visual Studio see https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/getting-started-extension.md#nightly--pre-release-instructions

**Comment by harshgangrade111** [comment]
@sibille  @vmadurga  Issue is still repro in the latest dev-nightly version. After pressing tab key screen reader announce buttons as group.

**Comment by sibille** [comment]
@harshgangrade111, just double checked and this works fine for me. Modified files are marked as "Button", and Narrator is saying "Button". Does this look correct to you or did we miss something? 
![image](https://user-images.githubusercontent.com/24698018/98688570-4a4d8800-236b-11eb-9706-a1ddfb4b1ca7.png)

Not sure why it's behaving different for you: Did you check in the context menu for the dev-nightly version?
You'll have to uninstall the production version 3.8 to make the context menu work correctly.
![image](https://user-images.githubusercontent.com/24698018/98688689-6a7d4700-236b-11eb-84e9-8051373c80d1.png)
 
You can see the version you are using at the right side bar:
![image](https://user-images.githubusercontent.com/24698018/98689576-59810580-236c-11eb-9d40-3ace500db28b.png)

**Comment by harshgangrade111** [comment]
@sibille Yes , it is defined as button but the issue here is that when we are navigating on it with the help of keyboard "Tab" key and screen reader is ON, Screenreader announces "Group" instead of button.

**Comment by sibille** [comment]
@harshgangrade111 , sorry, I'm not able to reproduce this. Narrator says "Button" now, not group.
![image](https://user-images.githubusercontent.com/24698018/98971784-e913e900-2511-11eb-8859-b4e66e5511a8.png)
Could you share a repro video, to see if we can reproduce the issue this way?

**Comment by harshgangrade111** [comment]
Issue does not repro now. Screen reader reads “Modified Files” button as a button not group . Hence closing this bug.
